modules/autoscale/main.tf

```diff
@@ -193,7 +193,9 @@ resource "google_pubsub_subscription_iam_member" "main" {
 #---------------------------------------------------------------------------------
 # The following resources are used for delicensing
 
-data "google_project" "this" {}
+data "google_project" "this" {
```

Comment:
if in `main.tf` we are using only `data.google_project.this.project_id`, then I propose to remove data source `google_project` and use only `var.project_id`.

If there are other use cases for `data.google_project`, then let's leave it as it is.

Reply:
Removed the data source entirely - left only project ID input variable .